ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/validator/CachingDamlLedgerStateReaderSpec.scala

```diff
@@ -68,9 +69,7 @@ class CachingDamlLedgerStateReaderSpec
 
   private def newInstance(damlLedgerStateReader: DamlLedgerStateReader)(
       implicit executionContext: ExecutionContext): CachingDamlLedgerStateReader = {
-    val cache = Caffeine
-      .newBuilder()
-      .build[DamlStateKey, DamlStateValue]
+    val cache = Cache.from[DamlStateKey, DamlStateValue](Configuration(1024))
```

Comment:
We have a parameter for this in other ledgers, would it make sense to use it here as well?

Reply:
Can you elaborate, please? This just creates a small cache to test the caching logic without having to mock the cache instance.

Reply:
please ignore, i didn't read filename